api/src/main/java/org/motechproject/nms/api/web/CallDetailsController.java

```diff
@@ -126,7 +126,7 @@ private void createCallDetailRecord(FrontLineWorker flw, CallDetailRecordRequest
             content.setEndTime(new DateTime(callContentRequest.getEndTime() * MILLISECONDS_PER_SECOND));
 
             if (service == Service.MOBILE_KUNJI) {
-                content.setMobileKunjiCardNumber(callContentRequest.getMkCardNumber());
+                content.setMobileKunjiCardNumber(callContentRequest.getMkCardCode());
```

Comment:
nitpick: should we also rename CallContent.mobileKunjiCardNumber to CallContent.mobileKunjiCardCode?